wallet/src/controller.rs

```diff
@@ -660,35 +660,38 @@ where
 fn create_error_response(e: Error) -> Response<Body> {
 	Response::builder()
 		.status(StatusCode::INTERNAL_SERVER_ERROR)
-		.header("access-control-allow-origin", "*")
+		.header(hyper::header::ACCESS_CONTROL_ALLOW_ORIGIN, "*")
 		.header(
-			"access-control-allow-headers",
+			hyper::header::ACCESS_CONTROL_ALLOW_HEADERS,
 			"Content-Type, Authorization",
 		)
+		.header(hyper::header::CONTENT_TYPE, "application/json")
```

Comment:
Is this safe to do? I suspect the error itself is not being serialized out as json in the body.

Reply:
Yes we noticed some content coming back that was just text, that must be from here. I'll wrap it up and see what you think

Reply:
This looks like more of a systemic issue. It looks like the API returns many (all?) errors as text without JSON wrapping. I can tackle this but it would take a bit more time. Perhaps we assume if the status code is OK then the content type must be JSON?

Reply:
That's probably correct yes. I'm not sure we promise _anything_ if the response is not an OK (and that's not a good thing).